public/js/main.js

```diff
@@ -42,3 +42,8 @@ document.getElementById('prev-button').onclick = function () {
 getEventsModule({
   date_from: new Date().toISOString().slice(0, 10)
 }, renderModule);
+
+document.getElementById('addPlaceIcon').addEventListener('click', function (e) {
+  document.getElementById('addPlaceContainer').classList.toggle('show');
+  document.getElementById('selectAllPlaces').classList.toggle('hide');
+});
```

Comment:
Woops, yes these do need to be in a separate file, forgot it was a different page 👍